server/channels/app/notification.go

```diff
@@ -1632,3 +1643,20 @@ func shouldChannelMemberNotifyCRT(userNotifyProps model.StringMap, channelMember
 
 	return
 }
+
+func ShouldAckWebsocketNotification(channelType model.ChannelType, userNotificationLevel, channelNotificationLevel string) bool {
+	if channelNotificationLevel == model.ChannelNotifyAll {
```

Comment:
There's very little traffic generated by one person even if they have the channel set to ALL. It would require a number of posts per second so high that no one would conceivably have it on for a channel that active, and even if they did, we can support up to about 10k ACKs per second before we start seeing timeouts.

I'd like to keep this in here to gather data unless we have a compelling reason (ie. performance testing) to remove it. In my performance testing, where I did have a user (myself) with ALL on, this wasn't causing an issue at all.